modules/AOR_Reports/aor_utils.php

```diff
@@ -93,31 +93,21 @@ function getDisplayForField($modulePath, $field, $reportModule)
     return array('field' => $fieldDisplay, 'type'=>$fieldType, 'module' => str_replace(' ', '&nbsp;', implode(' : ', $modulePathDisplay)));
 }
 
-function requestToUserParameters($reportBean = null)
+function requestToUserParameters()
 {
-    global $app_list_strings;
     $params = array();
-    if (!empty($_REQUEST['parameter_id'])) {
+    if(isset($_REQUEST['parameter_id']) && $_REQUEST['parameter_id']) {
```

Comment:
```!empty``` is preferred over ```isset(...) && ...```. Also PSR-2. 
```suggestion
    if (!empty($_REQUEST['parameter_id'])) {
```